rewrite-java/src/main/java/org/openrewrite/java/tree/J.java

```diff
@@ -1067,8 +1067,19 @@ public Case withBody(J body) {
             return getPadding().withBody(JRightPadded.withElement(this.body, body));
         }
 
+        @Nullable
+        JContainer<CaseLabel> labels;
```

Comment:
Changing this to a list of `JRightPadded` was mentioned but that brings along a load of changes. The expressions and statements fields are also JContainer so I feel like it's best to keep things in that line